spotify-client-api.el

```diff
@@ -80,7 +69,7 @@ globally relevant."
 (defvar *spotify-client-api-oauth2-ts*    nil
   "Unix timestamp in which the OAuth2 token was retrieved.
 This is used to manually refresh the token when it's about to expire.")
-(defvar *spotify-client-api-oauth2-token-directory* "~/.emacs.d/.cache/spotify"
+(defvar *spotify-client-api-oauth2-token-directory* (concat (file-name-as-directory user-emacs-directory) ".local/cache/spotify")
```

Comment:
@bnjmnt4n Thanks for this PR! From all the discussion I can find, I don't see any reference to `.local`. I was basing the token directory location on discussion I found [here](https://github.com/syl20bnr/spacemacs/issues/5947) and also [here](https://github.com/bbatsov/emacs-lisp-style-guide/issues/40). Was this intentional?

Reply:
Right, it appears that the `.local` path is a custom modification made internally by [doom-emacs](https://github.com/hlissner/doom-emacs). I'll remove the change.

Reply:
I'll go ahead and merge and make that tweak. Thanks! @bnjmnt4n